src/components/location/dto/list-locations.dto.ts

```diff
@@ -19,7 +19,7 @@ export abstract class LocationFilters {
   @Field(() => [String], {
     description: 'Filter to only these types of locations',
   })
-  readonly types?: Array<'country' | 'region' | 'zone'>;
+  readonly types?: Array<'region' | 'zone'>;
```

Comment:
I don't think this filter makes sense anymore.

Name can be removed as well, we have search for that.